Assignment/SampleData.cs

```diff
@@ -1,30 +1,105 @@
 ﻿using System;
+using System.IO;
+using System.Linq;
 using System.Collections.Generic;
 
+#pragma warning disable CA1065
+
 namespace Assignment
 {
     public class SampleData : ISampleData
     {
-        // 1.
-        public IEnumerable<string> CsvRows => throw new NotImplementedException();
+        private string _FileName { get; }

```

Comment:
```suggestion
        private string FileName { get; }
```
Properties use pascal.

Reply:
It's private, so I think I have the correct naming there, unless you think otherwise

Reply:
The access modifier doesn't matter.